src/Bicep.Local.Deploy/LocalDeploymentEngineHost.cs

```diff
@@ -140,19 +142,27 @@ protected override async Task<string> TryReadAsString(HttpContent content, bool
         }
     }
 
-    public override async Task<HttpResponseMessage> CallExtensibilityHost(
+    public override async Task<HttpResponseMessage> CallExtensibilityHostV2(
         HttpMethod requestMethod,
         Uri requestUri,
-        ExtensibilityOperationRequest request,
+        HttpContent content,
         AuthenticationToken extensibilityHostToken,
         CancellationToken cancellationToken)
     {
-        var response = await extensibilityHandler.CallExtensibilityHost(requestUri.Segments[^1], request, cancellationToken);
+        var extensionName = requestUri.Segments[^4].TrimEnd('/');
+        var extensionVersion = requestUri.Segments[^3].TrimEnd('/');
+        var method = requestUri.Segments[^1].TrimEnd('/');
```

Comment:
Worth creating a `record` to encapsulate this information and handle parsing?

Reply:
Would be even better to use a `readonly record struct` in this case.

Reply:
Is there any specific place we put these type of records? Or in the same class is ok?

Reply:
Created new file so it can be reference here and in local extensibility handler.